src/cmd-remote-build-container

```diff
@@ -239,6 +244,9 @@ Examples:
     parser.add_argument(
         '--git-sub-dir', default='', required=False,
         help='Git sub directory to use for container build')
+    parser.add_argument(
+        '--git-file', default='', required=False,
```

Comment:
minor: I think the name of the argument is confusing, `containerfile` is enough, with the help text as is IMO

Reply:
I kind of agree.. alternatively.. --contextdir= where you can specify the full path to the containerfile as an optoin.

Reply:
I want something with `git` in it to make it clear it's about the git repo. Maybe just `--git-containerfile`?

Reply:
OK, just went with `--git-containerfile` for now.